server_installation/topics/operator/installation.adoc

```diff
@@ -0,0 +1,68 @@
+
+=== Installing {project_operator} into OpenShift cluster
+
+There are ways of running {project_operator} in Kubernetes/OpenShift cluster:
+
+* Using Operator Lifecycle Manager (OLM)
+* Running manually
+
+==== Using Operator Lifecycle Manager
+
+In case of a newly installed OpenShift 4.x cluster, {project_operator} should already be available in
+OperatorHub tab of the OpenShift administration panel (as shown in the image below).
+
+image:{project_images}/operator-openshift-operatorhub.png[]
+
+If {project_operator} is not available, please navigate to https://operatorhub.io/operator/keycloak-operator, click install button and follow the instructions on screen.
```

Comment:
Maybe "click `Install` button"? (if "Install" is the name of the button)

Reply:
Fixed,